index.html

```diff
@@ -57,7 +58,7 @@ <h2 class="text-center">Who can come?</h2>
         <span class="glyphicon glyphicon-star text-yellow"></span>
       </p>
       <h2 class="text-center">How does it work?</h2>
-      <p>Grab a ticket when the meetup is announced. We'll book lanes at the alley and all you need to do is turn up and pay for your bowling. If you don't get a ticket we can't guarantee there'll be space on the lane.</p>
+      <p>Grab a ticket when the meetup is announced. We'll book lanes at the alley and all you need to do is turn up and pay for your bowling. Ifyou don't get a ticket or if you arrive after 7pm (when we start the bowling itself) we can't guarantee there'll be space on the lane.</p>
```

Comment:
I can't tell if the **Ifyou** has a space between the two words or not... @iteles